packages/lib-user/src/components/shared/ContentBox/components/ContentHeading.js

```diff
@@ -0,0 +1,37 @@
+import { SpacedText } from '@zooniverse/react-components'
+import { Heading } from 'grommet'
+import { node } from 'prop-types'
+import styled from 'styled-components'
+
+const StyledHeading = styled(Heading)`
+  font-size: 16px;
```

Comment:
```suggestion
  font-size: 1rem;
```

If someone increases the font size in their browser, we want this to increase too. Actually, you can use `rem` units in the `size` prop on Heading components if that's easier to read such as `size='1rem'`.

Reply:
Ah yes, just read that in the related nice article, thanks for flagging!